module2/homework/insensitivePalindrom/palindrome.cpp

```diff
@@ -0,0 +1,11 @@
+#include "palindrome.hpp"
+
+#include <algorithm>
+
+bool is_palindrome(const std::string& PalStr) {
```

Comment:
I do not know (maybe some more experienced C++ would know that) Does it make sense to make const ref in function signature, if we are making copy one line below?

Reply:
By doing so we do not modify original string PalStr, it makes a sense to me

Reply:
When you pass by value, then you need to use std::move. There is also string_view in C++17 which could operate with std:;string in cheap way.

Reply:
Good to know.Some objects are not moveable.